### PR TITLE
chore(docs content): rename ES to OpenSearch

### PR DESCRIPTION
**Pull request by siegerts**
_Notes:_

- only content in `src/pages` or `src/fragments`
- **ignore** content  in `docs/` (legacy)

_Description of changes:_

- [x] initial pass
- [x] URLS (searchable.mdx)
- [ ] instance types (searchable.mdx)
- [ ] redirects
- [x] cspell updates
- [ ] team reviews

cc @jakeburden @mauerbac 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/cli/graphql-transformer/config-params.mdx [hunk 1]
```diff
@@ -116,7 +116,7 @@ Follow these two steps when you need to rotate an API Key
 
 ## ElasticSearchInstanceCount
```
Comment (siegerts) [inline src/pages/cli/graphql-transformer/config-params.mdx:117]: @danielleadams Are any of these config name params changing?

**Review by siegerts — COMMENTED** [review]
~~@danielleadams - `ElasticSearchDomain` value changing?~~

File: src/pages/cli/graphql-transformer/searchable.mdx [hunk 2]
```diff
@@ -133,13 +133,13 @@ Here is a complete list of searchable operations per GraphQL type supported as o
 
 ### Backfill your Elasticsearch index from your DynamoDB table
 
-The following Python [script](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-elasticsearch-transformer/scripts/ddb_to_es.py) creates an event stream of your DynamoDB records and sends them to your Elasticsearch Index. This will help you backfill your data should you choose to add `@searchable` to your @model types at a later time.
+The following Python [script](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-elasticsearch-transformer/scripts/ddb_to_es.py) creates an event stream of your DynamoDB records and sends them to your OpenSearch Index. This will help you backfill your data should you choose to add `@searchable` to your @model types at a later time.
```
Comment (siegerts) [inline src/pages/cli/graphql-transformer/searchable.mdx:136]: Are the `graphql-elasticsearch-transformer` package name references changing (`opensearch`)?

File: src/pages/cli/graphql-transformer/config-params.mdx (hunk 1 above)
Comment (renebrandel) [reply src/pages/cli/graphql-transformer/config-params.mdx:117]: No.

File: src/pages/cli/graphql-transformer/searchable.mdx (hunk 2 above)
Comment (renebrandel) [reply src/pages/cli/graphql-transformer/searchable.mdx:136]: No.

**Review by cjihrig — APPROVED** [review]

**Comment by siegerts** [comment]
Closing in favor of https://github.com/aws-amplify/docs/pull/3424 against the current site structure.

**Pull request closed without merging**